action.yaml

```
@@ -0,0 +1,43 @@
+---
+name: 'Artifacts Setup'
+description: 'Set outputs to work with Artifacts Actions'
+inputs:
+  artifacts_url:
+    description: 'The url of Artifacts'
+    required: false
+    # TODO: discuss the fact that secrets cannot be set
+    # as defaults on inputs.
+    default: 'https://artifacts.scality.net'
```

As this thing is in public, remove unwanted default value than return to scality ?

---

We need to discuss this one together, you are right, I was hoping we could find a way to define those as the following:
```suggestion
    default: '${{ secret.ARTIFACTS_URL }}'
```

But secrets are not supported as default value on the input :cry: 

So I think we need to define this as "required" with no default value and make the user set the secret on the other end.